deployment-aws/puppet/manifests/site.pp

```diff
@@ -12,6 +12,7 @@
     database_superuser_password    => $facts['database_superuser_password'],
     database_user_password         => $facts['database_user_password'],
     data_volume_device_name        => $facts['data_volume_device_name'],
+    deployment_stack               => $facts['deployment_stack'],
```

Comment:
I'd reuse another variable for namespace like `$mevapi::cloudwatch_log_group` or `$mevapi::backend_domain` instead of adding yet another parameter.

Reply:
there are other places (unrelated to this PR) where i think naming/namespacing would be improved by appending a stack identifier, so I see it as being potentially useful.